styles/myStory.css

```diff
@@ -0,0 +1,86 @@
+.overlay-container {
+  width: 100%;
+  height: 100vh;
+  position: relative;
+}
+.overlay-content {
+  max-width: 1500px;
+  margin: 0 auto;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  padding-left: 20px;
+  height: 90%;
+}
+.overlay-text {
+  position: absolute;
+  color: white;
+  font-family: var(--paragraph-sans);
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.overlay-text p {
+  margin-bottom: 50px;
+  margin: 0px;
+  text-transform: uppercase;
+  text-align: center;
+}
+.text-header {
+  font-size: 68px;
+}
+.text-description {
+  padding-top: 20px;
+  font-size: 16px;
+}
+.color-overlay {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background-color: rgba(0, 0, 0, 0.3);
+}
+.background-image {
```

Comment:
for this use case it's much better to use the background: url() to render the background image instead of creating another div

Reply:
thansk for the suggestion, I removed the div and added the background-image:URL() together with background-size: cover and the image is rendered just well.